### PR TITLE
Add the `disableNativeSpellChecker` config parameter to the bundle

### PR DESCRIPTION
**Pull request by martin-damien**
This pull request allow users to set the value of `disableNativeSpellChecker` as described in http://docs.ckeditor.com/#!/guide/dev_spellcheck

### CODE REVIEW

File: DependencyInjection/Configuration.php [hunk 1]
```diff
@@ -76,6 +76,11 @@ public function getConfigTreeBuilder()
                     ->info("The user interface language localization to use.")
                     ->example("en-au")
                 ->end()
+                ->scalarNode('disable_native_spell_checker')
+                    ->defaultNull()
+                    ->info("Disabled the browser's spell checker.")
```
Comment (trsteel88) [inline DependencyInjection/Configuration.php]: Please change this to match the Ckeditor documentation: "Disables the built-in spell checker if the browser provides one."

File: DependencyInjection/Configuration.php [hunk 2]
```diff
@@ -76,6 +76,11 @@ public function getConfigTreeBuilder()
                     ->info("The user interface language localization to use.")
                     ->example("en-au")
                 ->end()
+                ->scalarNode('disable_native_spell_checker')
```
Comment (trsteel88) [inline DependencyInjection/Configuration.php]: This should be a booleanNode

File: DependencyInjection/Configuration.php [hunk 3]
```diff
@@ -76,6 +76,11 @@ public function getConfigTreeBuilder()
...
+                    ->defaultNull()
+                    ->info("Disabled the browser's spell checker.")
+                    ->example('true')
```
Comment (trsteel88) [inline DependencyInjection/Configuration.php]: Not necessary for booleanNode

File: Resources/views/Form/ckeditor_widget.html.twig [hunk 4]
```diff
@@ -39,6 +39,9 @@
             {% if language is not null %}
                 language: '{{ language }}',
             {% endif %}
+            {% if disable_native_spell_checker is not null %}
+                disableNativeSpellChecker: {% if disable_native_spell_checker %}true{% else %}false{% endif %},
```
Comment (trsteel88) [inline Resources/views/Form/ckeditor_widget.html.twig]: Change to:

```twig
disableNativeSpellChecker: {{ disable_native_spell_checker }},
```

**Comment by trsteel88** [comment]
Can you please write some tests for this. You can use the following tests as examples:

[Test 1](https://github.com/trsteel88/TrsteelCkeditorBundle/blob/master/Tests/Form/CkeditorTypeTest.php#L200) - Obviously this should default to null for your parameter though.

[Test 2](https://github.com/trsteel88/TrsteelCkeditorBundle/blob/master/Tests/Form/CkeditorTypeTest.php#L212)

**Comment by trsteel88** [comment]
This has been completed and is available on (v1.8.1)[https://github.com/trsteel88/TrsteelCkeditorBundle/releases/tag/v1.8.1]

**Comment by martin-damien** [comment]
Thank you very much.

**Pull request closed without merging**